fe1-web/features/social/network/ChirpHandler.ts

```diff
@@ -1,11 +1,12 @@
 import { ExtendedMessage } from 'model/network/method/message';
 import { ActionType, MessageRegistry, ObjectType } from 'model/network/method/message/data';
 import {
-  addChirp, deleteChirp, dispatch, getStore, makeCurrentLao,
+  dispatch, getStore, makeCurrentLao,
 } from 'store';
 
 import { Chirp } from '../objects';
 import { AddChirp, DeleteChirp } from './messages/chirp';
+import { addChirp, deleteChirp } from '../reducer/SocialReducer';
```

Comment:
at some point, we might need to find a better naming differentiation between reducer actions and messages 😅

Reply:
It would be a good idea